ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml

```diff
@@ -540,6 +540,7 @@ tests:
                   -e2e.node-pool-replicas=5 \
                   -e2e.base-domain="fake" \
                   -e2e.kubevirt-container-disk-image="${RHCOS_CD_IMAGE}" \
+                  --e2e.control-plane-operator-image=${HYPERSHIFT_RELEASE_LATEST} \
```

Comment:
Need the relevant dependencies, otherwise this won't work and just pass an empty string
/hold

Reply:
I thought that's declared here:
https://github.com/openshift/release/pull/27060/files#diff-8de98e1c6b45dfcd057dd3229c06140b0cbc08dd4d9b54629026fe849c7dfaeeL549

Reply:
the test already uses that image to [install hypershift](https://github.com/openshift/release/pull/27060/files#diff-8de98e1c6b45dfcd057dd3229c06140b0cbc08dd4d9b54629026fe849c7dfaeeR527), it's just now specifying the same image for the cpo

Reply:
You are right, my bad
/hold cancel